packages/manager/modules/nasha/src/components/partition/zfs-options/zfs-options.constants.js

```diff
@@ -0,0 +1,16 @@
+export const OPTIONS_RESOLVE = {
+  options: /* @ngInject */ ($q, $http, partitionApiUrl, prepareZfsOptions) =>
+    $http
+      .get(`${partitionApiUrl}/options`)
+      .catch((error) =>
+        error.status === 404 ? { data: prepareZfsOptions() } : $q.reject(error),
+      )
+      .then(({ data }) => prepareZfsOptions(data)),
+};
+
+export const TRANSLATE = 'nasha_components_partition_zfs_options';
```

Comment:
```suggestion
export const TRANSLATE_PREFIX = 'nasha_components_partition_zfs_options';
```

Reply:
:+1: 
This has been done here : https://github.com/ovh/manager/pull/7495/files#diff-7915a2ac644befb82d1c7933cf78c5f0261701d9a5638cdfee26494028c46ea9R1